src/Modal/style.css

```diff
@@ -100,6 +100,10 @@
     &:nth-last-child(2) {
         padding-bottom: var(--inner-gap);
     }
+
+    &.is-fullwidth {
+        padding: 0 0 0 0;
```

Comment:
I think the fullwidth should remove only the padding left and right. But not top and bottom, because these paddings depends on other things (if the element is the first-child or last-child, for example). So I'd change it for:

```css
&.is-fullwidth {
    padding-left: 0;
    padding-right: 0;
}
```